openslides_backend/action/actions/committee/update.py

```diff
@@ -34,20 +38,56 @@ class CommitteeUpdateAction(CommitteeCommonCreateUpdateMixin, UpdateAction):
             "forward_to_committee_ids",
             "receive_forwardings_from_committee_ids",
             "organization_tag_ids",
-        ]
+        ],
+        additional_optional_fields={"manager_ids": id_list_schema},
     )
 
-    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
-        instance = super().update_instance(instance)
-        if instance.get("template_meeting_id"):
-            self.check_meeting_in_committee(
-                instance["template_meeting_id"], instance["id"]
-            )
-        if instance.get("default_meeting_id"):
-            self.check_meeting_in_committee(
-                instance["default_meeting_id"], instance["id"]
-            )
-        return instance
+    def get_updated_instances(self, action_data: ActionData) -> ActionData:
+        for instance in action_data:
+            instance = super().update_instance(instance)
```

Comment:
Do not call `update_instance` inside `get_updated_instances`, is called by `perform` automatically afterwards. Also, I do not understand why you can't just put all your code in `update_instance`, if you `pop` the `manager_ids` before returning, everything should work nicely.

If not, at least you can leave the previous checks for `*_meeting_id` in `update_instance` since they belong there.

Reply:
If you got the payload `{"id": X, "manager_ids": [X, Y]}` and pop the manager_ids then the ds will return a error.